src/pages/Home.js

```diff
@@ -1,21 +1,69 @@
+import React, { useEffect, useRef } from 'react';
 import { useQuery } from 'react-query';
-import { Link } from 'react-router-dom';
+import { json, Link } from 'react-router-dom';
+import { useState } from 'react';
 
 const new_query = `
 {
   foodspots {
     edges {
       node {
         id
-        title
         link
+        title
+        foodspot_daten {
+          adresse
+          bewertung
+          dineIn
+          delivery
+          homepage
+          mapsLink
+          offnungszeiten
+          placeId
+          preisklasse
+          takeout
+          telefon
+        }
       }
     }
   }
 }
 `;
 
 function Home() {
+  let [filter, setFilter] = useState({
+    delivery: false,
+    dineIn: false,
+    takeout: false,
+    preisklasse: 0,
+  });
+
+  let [isFilterActive, setIsFilterActive] = useState();
+  let [resultArray, setResultArray] = useState([]);
+
+  let ref = useRef([]);
+  let radioBtns = ref.current.elements;
+
+  let setFilterOff = () => {
```

Comment:
Fehleranfällig, wenn filter dazukommen muss diese Funktion immer um den neuen Filter erweitert werden --> sollte in einer Schleife durchlaufen werden.

Reply:
vorletzte commit